lib/onfido/resource.rb

```diff
@@ -39,7 +39,8 @@ def make_request(options)
 
       response = RestClient::Request.execute(request_options)
 
-      parse(response)
+      #response should be parsed only when there is a response expected
+      parse(response) if response.code != 204 #no_content
```

Comment:
I'm :+1: on @abousamra's approach here, although I think I'd prefer `unless response.code == 204` rather than `if response.code != 204`, semantically.

As an aside, what response do you get to `PUT` requests like resuming a check? Do you get content back? I'm guessing it returned the resumed resource?